### PR TITLE
Start checklist for checking maintainance mode

### PR DESCRIPTION
**Pull request by jwhitlock**
Related to issue #409, start on a checklist for manual verification of maintenance mode in AWS.  I think a related checklist is needed for provisioning MM, which has been started in that issue, and may be useful after the prototype.

### CODE REVIEW

**Comment by metadave** [comment]
@jwhitlock can you move these to a (new) `apps/mdn/mdn-aws/docs` directory?

**Comment by jwhitlock** [comment]
I've moved the checklist to ``apps/mdn/mdn-aws/docs/mdn_maintenance_mode_checklist.md``, and went ahead with the current URL https://mdn-mm.moz.works in the docs.

I'm not sure how many manual steps are needed to get a deployment into maintenance mode - that may deserve it's own documentation.

My next step is to flesh out the manual testing section, by surveying the endpoints in the Kuma code.

File: apps/mdn/mdn-aws/docs/mdn_maintenance_mode_checklist.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+* Interactive Examples: https://interactive-examples.mdn.moz.works
+
+# Preparing for Maintenance Mode
+
+If you want a public-facing site, you'll need to generate
+content before putting the site in Maintenance Mode. The
+full steps are in *todo*, but here's the quick version:
+
+* [ ] Load a recent (anonymized) backup
+* [ ] Enable GitHub auth (see [Enable GitHub Auth](https://kuma.readthedocs.io/en/latest/installation.html#enable-github-auth-optional))
```
Comment (safwanrahman) [inline apps/mdn/mdn-aws/docs/mdn_maintenance_mode_checklist.md]: In maintenance mode, user cant log in. So Github auth is required?

Comment (jwhitlock) [reply apps/mdn/mdn-aws/docs/mdn_maintenance_mode_checklist.md]: I think you need to be logged in and use the Django admin to generate the search index, so you'll need to do some things before switching to MM.

Comment (safwanrahman) [reply apps/mdn/mdn-aws/docs/mdn_maintenance_mode_checklist.md]: In the anonymized databse, the email is anonymized also. So if any admin tries to log in with github, he will be unable to do.
Moreover, I think admin should able to login with username password in the `/admin` url!

**Pull request merged**